packages/react-data-grid-examples/src/scripts/example29-descendingFirstSortable.js

```diff
@@ -0,0 +1,116 @@
+const ReactDataGrid = require('react-data-grid');
```

Comment:
Prefer `ES6 import/export` over `require/exports`. We are planning to migrate to `ES6 modules` soon